src/AcceptanceTests/OldTests/When_sending_an_oversized_message_from_a_transaction_scope.cs

```diff
@@ -40,7 +40,8 @@ class MyEndpoint : EndpointConfigurationBuilder
         {
             public MyEndpoint()
             {
-                EndpointSetup<DefaultServer>(c => c.ConfigureAzureServiceBus().Queues().MaxDeliveryCount(1));
+                // set delivery count to 1
```

Comment:
The comment is disturbing. Can we remove it? It doesn't help me because I see the number zero, then I see the delivery count mentioned to be one and then I have to mentally map those two things which doesn't help me in the context of an acceptance test.

Reply:
"disturbing" is an interesting choice of word.
When I will see recoverability with immediate retries set to zero in a few months, will not immediately recall that it will cause `MaxDeliveryCount` to be set to 1. Yet this is intentional in this test to eliminate re-processing.
How about " // Disable immediate retries to have max delivery count of 1" ?

Reply:
Why do we even need a comment? In an acceptancetest I try as much as I can to use lingo of the higher level and not the lower level.

Reply:
More than happy for suggestions.

Reply:
Here is my suggestion;how about we drop the comment?

Reply:
@danielmarbach when I read `When_sending_an_oversized_message_from_a_transaction_scope Should_throw_message_too_large_exception` and look at immediate retries set to zero, it will be hard to recall the reason immediate retries were disabled. You've provided a good suggestion to name things better to help with the context. Now you're saying forget it all and just drop the comment. I'll drop the comment, but this feels wrong. We haven't provided the context nor explanation _why_ immediate retries are disabled.

Reply:
I'm providing input here, you are the master of this PR, feel free to ignore my suggestions. I'll remove my changes request